include/zephyr/net/wifi_mgmt.h

```diff
@@ -115,6 +115,7 @@ enum net_event_wifi_cmd {
 	NET_EVENT_WIFI_CMD_TWT_SLEEP_STATE,
 	NET_EVENT_WIFI_CMD_RAW_SCAN_RESULT,
 	NET_EVENT_WIFI_CMD_DISCONNECT_COMPLETE,
+	NET_EVENT_WIFI_CMD_CONNECT_STATUS,
```

Comment:
The events in WPA supplicant span more than connection phase, do we plan to limit them to connection to map to these events? or should we have `NET_EVENT_WIFI_NM_EVENT` with `NM` specific event body?

Reply:
Done